deploy-manage/security/secure-your-cluster-deployment.md

```diff
@@ -5,11 +5,13 @@ applies_to:
     eck: all
     ece: all
     ess: all
+mapped_pages:
+  - https://www.elastic.co/guide/en/elasticsearch/reference/current/manually-configure-security.html
```

Comment:
I think this mapping might be better over here

https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/941/deploy-manage/security/self-setup